apple-codesign/src/lib.rs

```diff
@@ -111,7 +111,7 @@
 
 mod apple_certificates;
 pub use apple_certificates::*;
-pub mod app_store_connect;
+pub use app_store_connect;
```

Comment:
I have a soft preference for leaving this as it was before.

Reply:
don't quite follow this one. it's reexporting `app_store_connect` which has moved to a different crate to avoid breakage. we probably need to reexport at least the UnifiedApiKey.

Reply:
And I question the point in re-exporting an external crate completely. The dependency is there one way or another. IMO re-exporting just conflates crate boundaries and makes it harder to understand crate boundaries. There are exceptions to this, such as exporting limited, likely used symbols for convenience. But in the case of exporting a full crate, it isn't worth it IMO.

Reply:
ok